HC.Tomek/HC.Tomek.Js.Ex3/containers.js

```diff
@@ -0,0 +1,122 @@
+function startPage() {
+    document.getElementById("quiz").style.display = "none";
+    document.getElementById("result").style.display = "none";
+    var startContainer = document.getElementById("start");
+    var startButton = document.createElement("button");
+    startButton.innerHTML = "Start";
+    startContainer.appendChild(startButton);
+    startButton.addEventListener("click", function () {
+        document.getElementById("start").style.display = "none";
+        document.getElementById("quiz").style.display = "";
+        quizPage();
+    })
+}
+startPage();
+
+function quizPage() {
+    var quizContainer = document.getElementById("quiz");
+    var questionNumber = 3;
+    var maximumAnswers = 4;
+    var correctAnswers = 0;
+    var currentQuestionNumber = 0;
+    var questionText = new Array(questionNumber);
+    
+    var questionAnswerRadio = new Array(questionNumber);
```

Comment:
w JS nie trzeba podawać rozmiaru tablicy przy inicjalizacji zmiennej. Poza tym dobra praktyka to stosowanie literałów tzn zamiast new Object używamy { } a zamiast new Array używamy [ ] np:
var x = { };
var y = [ ];